pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -324,6 +324,8 @@ func (k *Kubernetes) InitDS(name string, service kobject.ServiceConfig) *extensi
 
 func (k *Kubernetes) initIngress(name string, service kobject.ServiceConfig, port int32) *extensions.Ingress {
 
+	hosts := strings.Split(service.ExposeService, ";")
```

Comment:
We should also use some form of checking (making sure spaces are removed, that someone doesn't add: `,,` for example..